tests/Collection/CollectionFunctionalTest.php

```diff
@@ -13,6 +13,8 @@
 use MongoDB\Exception\UnsupportedException;
 use MongoDB\MapReduceResult;
 use MongoDB\Operation\Count;
+use MongoDB\Operation\DropCollection;
+use MongoDB\Operation\FindOne;
```

Comment:
I was using `DropCollection` and `FindOne` for the renamed collection in `testRename()`, so I don't think I'm able to use the Collection class methods in this case right?

Reply:
In a later comment, I suggest creating a Collection instance for the renamed class. That would be taboo in the Operation test classes since those shouldn't concern themselves with the Collection class, but it shouldn't be a concern in CollectionFunctionalTest since we're already calling Collection methods elsewhere in this class.